.travis.yml

```diff
@@ -29,6 +35,26 @@ install:
       mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/openebs;
       cd $GOPATH/src/github.com/openebs/cstor-operators;
     fi
+  - if [ "$TRAVIS_CPU_ARCH" == "amd64" ]; then
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+    sudo chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    echo "Kubectl Client Version..."
+    kubectl version --client
+
+    echo "Downloading Minikube Binary..."
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+    && chmod +x minikube
+
+    sudo mkdir -p /usr/local/bin/
+    sudo install minikube /usr/local/bin/
+    sudo chown -R travis /home/travis/.minikube/;
+    echo "Minikube Version..."
+    minikube version
+    sudo minikube start --vm-driver=none --kubernetes-version=v1.16.0
```

Comment:
replace with 
```sh
# Download kubectl, which is a requirement for using minikube.
- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.18.1/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
# Download minikube.
- curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.1/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
- mkdir -p $HOME/.kube $HOME/.minikube
- touch $KUBECONFIG
- sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=v1.18.1
- minikube update-context --profile=minikube
- "sudo chown -R travis: /home/travis/.minikube/"
```

Reply:
What is the requirement?

Reply:
Is the travis not kicking in due to this ?